src/utils/containsValidUrl.ts

```diff
@@ -0,0 +1,8 @@
+export default function containsValidUrl(content: string): boolean {
```

Comment:
Maybe use this package for URL validation instead? https://www.npmjs.com/package/valid-url